app/src/main/java/com/training/cinematic/Adapter/MovieAdpater.java

```diff
@@ -0,0 +1,48 @@
+package com.training.cinematic.Adapter;
+
+import android.content.Context;
+import android.graphics.Movie;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.training.cinematic.R;
+import com.training.cinematic.activity.RecyclerViewHolders;
+
+import java.util.List;
+
+/**
+ * Created by dhruvisha on 5/26/2018.
+ */
+
+public class MovieAdpater extends RecyclerView.Adapter<RecyclerViewHolders> {
```

Comment:
You can embed holder class in adapter itself. e.g ... extends RecyclerView.Adapter<MovieAdapter.MovieHolder>